library/ecp.c

```diff
@@ -927,7 +927,7 @@ int mbedtls_ecp_point_read_binary(const mbedtls_ecp_group *grp,
     size_t plen;
     ECP_VALIDATE_RET(grp != NULL);
     ECP_VALIDATE_RET(pt  != NULL);
-    ECP_VALIDATE_RET(buf != NULL);
+    ECP_VALIDATE_RET(ilen == 0 || buf != NULL);
```

Comment:
Is that a minor bugix? Or does it align the backport to what is expected on the original PR?

Reply:
Technically it's a bug fix. It was needed to pass the tests (return the expected error rather than asserting when parameter checks are enabled). Should it have a changelog entry? It's only changing the reaction to an error case.

It doesn't apply to the original because we removed the VALIDATE stuff in 3.0.

Reply:
I agree it's technically a bugfix, but don't think it's important enough to deserve a ChangeLog entry (only changing the way the error is caught, and in a config that's marginal enough that we removed it in 3.0).

Reply:
Yes as long as it was intended and it is not applicable in 3.X I am in agreement with @mpg. It does not warrant a Changelog entry.